src/main/java/org/jabref/gui/preferences/customization/CustomizationTabViewModel.java

```diff
@@ -23,19 +47,58 @@ public CustomizationTabViewModel(DialogService dialogService, PreferencesService
         this.dialogService = dialogService;
         this.preferencesService = preferencesService;
         this.initialDOIPreferences = preferencesService.getDOIPreferences();
+        initApiKeyNameUrl();
+    }
+
+    private void initApiKeyNameUrl() {
+        // Springer query using the parameter 'q=doi:10.1007/s11276-008-0131-4s=1' will respond faster
+        API_KEY_NAME_URL.put("Springer", "https://api.springernature.com/meta/v1/json?q=doi:10.1007/s11276-008-0131-4s=1&p=1&api_key=");
+        API_KEY_NAME_URL.put("IEEEXplore", "https://ieeexploreapi.ieee.org/api/v1/search/articles?max_records=0&apikey=");
+    }
```

Comment:
Can't those be retrieved directly from the fetchers? (Open-Closed-Principle of clean coding: https://en.wikipedia.org/wiki/Open%E2%80%93closed_principle)

Reply:
fixed.